tkg/kind/client_test.go

```diff
@@ -19,6 +19,9 @@ import (
 	"github.com/vmware-tanzu/tanzu-framework/tkg/constants"
 	"github.com/vmware-tanzu/tanzu-framework/tkg/kind"
 	"github.com/vmware-tanzu/tanzu-framework/tkg/utils"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
```

Comment:
Was this intended? (given that there is no other diff in this file)

Reply:
Yup. For some reason it was missing and this file was otherwise broken.

Reply:
Interesting that that didn't surface in some way

Reply:
Yeah, i need to take a look at why.